packages/components/organisms/f-header/src/tenants/en-GB.js

```diff
@@ -69,7 +69,7 @@ export default {
     },
     deliveryEnquiry: {
         text: 'Deliver with Just Eat',
-        url: 'https://just-eat.co.uk/info/delivering-with-just-eat',
+        url: 'https://couriers.just-eat.co.uk/application',
         gtm: 'click_courier_signup'
```

Comment:
is it worth having a test to confirm it goes to the right place? something like https://github.com/justeattakeaway/fozzie-components/blob/master/packages/components/pages/f-registration/test/component/f-registration.component.spec.js#L34

Reply:
Discussed this with Lizzie. The combination of low priority for testing, low risk of change (this is always manually tested when the links are updated) and difficulties running the tests on Windows machines means I won't write a new test for this today.